src/mlpack/methods/pca/pca.hpp

```diff
@@ -68,6 +68,14 @@ class PCA
   void Apply(const arma::mat& data,
              arma::mat& transformedData,
              arma::vec& eigVal);
+/**
+   * Apply Principal Component Analysis to the provided data set. It is safe
+   * to pass the same matrix reference for both data and transformedData.
```

Comment:
This one is missing an empty line between the method description and the parameter description.